stable/dynamic-gateway-service/templates/statefulset.yaml

```diff
@@ -96,6 +96,31 @@ spec:
                   release: {{ .Release.Name }}
 {{- end }}
       initContainers:
+        - name: dpm-check
+          image: "{{ .Values.datapower.busybox.repository }}:{{ .Values.datapower.busybox.tag }}"
+          imagePullPolicy: {{ .Values.datapower.busybox.pullPolicy }} 
+          env:
+            - name: DPM_FQDN
+              value: "{{ .Release.Name }}-dpm-svc.{{ .Release.Namespace }}.svc.cluster.local"
+            - name: DPM_LIVENESS_PORT
+              value: {{ .Values.datapowerMonitor.livenessPort | default 8080 | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              export DPM_LIVENESS_URL="http://$DPM_FQDN:$DPM_LIVENESS_PORT/"
```

Comment:
We should consider putting this in as a script file in `static` instead of writing out all the code in the statefulset. The only reason this second init container makes sense is to separate the two chunks of code. This can easily be done with ConfigMaps passing in static scripts in a volume. If we were to need yet another bit of init functionality, we wouldn't want to add a third init container, I think.